javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java

```diff
@@ -0,0 +1,982 @@
+            System.err.println(e.awsErrorDetails().errorMessage());
+            System.exit(1);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+    // snippet-end:[cloudwatch.java2.scenario.del.anomalydetector.main]
+
+    // snippet-start:[cloudwatch.java2.scenario.del.alarm.main]
+    public static void deleteCWAlarm(CloudWatchClient cw, String alarmName) {
+        try {
+            DeleteAlarmsRequest request = DeleteAlarmsRequest.builder()
+                .alarmNames(alarmName)
+                .build();
+
+            cw.deleteAlarms(request);
+            System.out.println("Successfully deleted alarm " +alarmName);
+
+        } catch (CloudWatchException e) {
+            System.err.println(e.awsErrorDetails().errorMessage());
+            System.exit(1);
+        }
+    }
+    // snippet-end:[cloudwatch.java2.scenario.del.alarm.main]
+
+    // snippet-start:[cloudwatch.java2.scenario.del.dashboard.main]
+    public static void deleteDashboard(CloudWatchClient cw, String dashboardName) {
+        try {
+            DeleteDashboardsRequest dashboardsRequest = DeleteDashboardsRequest.builder()
+                .dashboardNames(dashboardName)
+                .build();
+            cw.deleteDashboards(dashboardsRequest);
+            System.out.println(dashboardName + " was successfully deleted.");
+
+        } catch (CloudWatchException e) {
+            System.err.println(e.getMessage());
+            System.exit(1);
+        }
+    }
+    // snippet-end:[cloudwatch.java2.scenario.del.dashboard.main]
+
+    // snippet-start:[cloudwatch.java2.scenario.get.metric.image.main]
+    public static void getAndOpenMetricImage(CloudWatchClient cw, String fileName) {
+        System.out.println("Getting Image data for custom metric.");
+        try {
+              String myJSON = "{\n" +
+                "  \"title\": \"Example Metric Graph\",\n" +
+                "  \"view\": \"timeSeries\",\n" +
+                "  \"stacked \": false,\n" +
+                "  \"period\": 10,\n" +
+                "  \"width\": 1400,\n" +
+                "  \"height\": 600,\n" +
+                "  \"metrics\": [\n" +
+                "    [\n" +
+                "      \"AWS/Billing\",\n" +
+                "      \"EstimatedCharges\",\n" +
+                "      \"Currency\",\n" +
+                "      \"USD\"\n" +
+                "    ]\n" +
+                "  ]\n" +
+                "}";
+
+            GetMetricWidgetImageRequest imageRequest = GetMetricWidgetImageRequest.builder()
+                .metricWidget(myJSON)
+                .build();
+
+            GetMetricWidgetImageResponse response = cw.getMetricWidgetImage(imageRequest);
+            SdkBytes sdkBytes = response.metricWidgetImage();
+            byte[] bytes = sdkBytes.asByteArray();
+            File outputFile = new File(fileName);
+            try (FileOutputStream outputStream = new FileOutputStream(outputFile)) {

```

Comment:
Could you tell the user in the output where the file is saved, or that the file is saved?

Reply:
Its described in the notes at start of the example